src/main/java/com/googlecode/lanterna/gui2/MultiWindowTextGUI.java

```diff
@@ -356,6 +362,9 @@ public synchronized Interactable getFocusedInteractable() {
 
     @Override
     public synchronized boolean handleInput(KeyStroke keyStroke) {
+        ifMouseDownPossiblyChangeActiveWindow(keyStroke);
+        ifMouseDownPossiblyStartTitleDrag(keyStroke);
+        ifMouseDragPossiblyMoveWindow(keyStroke);
```

Comment:
Shouldn't you stop event propagation here if any of the above matched the event?

Reply:
Nope, the window may not be focused, and then it can become focused and handle the event.
For the drag, it could have some logic flags added for that, but not sure the benefit, since the drag will only occur when the mouse event misses all others anyway.  It would produce more code for the same effect.